feat(code-mappings): Add note about automatic code mapping derivation (#5956)
getsentry/sentry-docs: src/docs/product/integrations/source-code-mgmt/github/index.mdx

</Note>

> <Note>
> Sentry will automatically try to set this up for your organization for Python and JavaScript projects for organizations with the Github integration installed. You are still welcome to add code mappings youself, however, we will still try to do so. Support for other languages and other source code integrations will follow.
> </Note>

Reviewer: ```suggestion
<Note>

Sentry will automatically try to set up code mappings on Python and JavaScript projects for organizations with the GitHub integration installed. However, you can still manually add code mappings yourself. Support for other languages and other source code integrations are planned.

</Note>
```

Stack trace linking takes you from a file in your Sentry stack trace to that same file in your source code. If you have commit tracking set up in Sentry, we can take you to the exact version (using the commit associated with the event) of the source code when the error occurred. Otherwise we'll link you to the current state of the source code (using the default **branch**).

> #### Manual set up

Reviewer: ```suggestion
#### Manual Setup
```

1. Navigate to **Settings > Integrations > GitHub > Configurations**.

> 2. Click the "Configure" button next to your GitHub Instance.

Author: Why do some pages just have a `1.`? What's the standard that should be followed?

        - ex. For example, everything after the branch (`main`) matches the file path of `code.py` using a source code path of `https://github.com/MeredithAnya/testing/blob/main/code.py` so you don't need to set a **stack trace root** and **source code root**.

     2. If the filename in your Sentry stack trace frame doesn't match the path to your source code, you will need to replace the **stack_root** part of the filename with your **source_root** to make the filename match the source code path.
        - ex. For example, to get `src/code.py` to match `code.py` when the source code path is `https://github.com/MeredithAnya/testing/blob/main/code.py`, change the **stack trace root** to be set as `src/`, and leave **source code root** empty.

> #### Stacktrace modal

Author: This information is out of date in [the issue owners page](https://docs.sentry.io/product/issues/issue-owners/) (see screenshot).

In my follow up PR, I will deal with updating it.

<img width="460" alt="image" src="https://user-images.githubusercontent.com/44410/207358143-80200cfc-2b00-483c-8ac4-6eb50e77c32a.png">

> #### Stacktrace modal

Reviewer: Since this is information about an interaction you're going to have on the Issue Details page, this should probably be in that content, rather than here. But let's discuss offline. I'm not clear on this strategy.
